css/style.css

```diff
@@ -0,0 +1,177 @@
+@import url('https://fonts.googleapis.com/css2?family=Abhaya+Libre:wght@400;500;700&family=Roboto:wght@300;400;500;700&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+    font-size: 14px;
+    font-weight: 400;
+}
+
+a {
+    color: #111111;
+    text-decoration: none;
+}
+
+.container {
+    max-width: 960px;
+    margin: 0px auto;
+}
+
+.postContain {
+    position: relative;
+    padding: 35px 0px 21px 40px;
+}
+
+.postContain::after {
+    content: "";
+    width: 1px;
+    height: 100%;
+    position: absolute;
+    left: 0;
+    top: 0;
+    background: #6A6A6A;
+}
+
+.post {
+    margin-top: 30px;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.postImg img {
+    width: 100%;
+    display: block;
+    border-radius: 7px;
+}
+
+.postImg {
+    width: 271px;
+}
+
+.postMain {
+    padding-left: 32px;
+    box-sizing: border-box;
+    flex-basis: 50%;
+    flex-grow: 1;
+}
+
+.postCategory {
+    margin-top: 0px;
+    padding: 5px 10px;
+    background: #262626;
+    display: inline-block;
+    color: #FBFBFB;
+    font-size: 11px;
+    margin-top: 10px;
+}
+
+.postTitle {
+    display: block;
+    margin-top: 13px;
+    font-size: 26px;
+    font-weight: 700;
+}
+
+.postDescription {
+    margin-top: 6px;
+    font-size: 17px;
+    color: #333333;
+}
+
+.postInfoShare {
+    display: flex;
+    padding: 5px 0px;
+    margin-top: 10px;
+}
+
+.postInfo {
+    padding-right: 10px;
+}
+
+.postShare {
+    padding-left: 10px;
+    border-left: 1px solid #000000;
+    text-decoration: underline;
+}
+
+.pagination {
+    margin-top: 30px;
+}
+
+.paginate {
+    display: flex;
+    list-style: none;
+}
+
+.paginate a {
+    display: block;
+    margin: 0px 5px;
+    font-size: 20px;
+}
+
+.paginateContain {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.arrowPaginate {
+    margin: 0px 10px;
+    cursor: pointer;
+    padding: 5px 8px;
+}
+
+.txtTitle {
+    color: #262626;
+    position: absolute;
+    left: -125px;
+    top: 10px;
+    font-size: 21px;
+}
+
+.arrowStories {
+    display: inline-block;
+    width: 25px;
+    height: 10px;
+}
+
+.arrowStories img {
+    width: 100%;
+    display: block;
+    transform: rotate(-90deg);
+}
+
+@media only screen and (max-width: 757px) {
```

Comment:
<img width="769" alt="Screen Shot 2020-08-28 at 11 52 16 PM" src="https://user-images.githubusercontent.com/56341874/91592972-89ce2000-e989-11ea-92ab-2ecc719d68b4.png">
Take Bootstrap responsive size for reference. Your responsive size seems a little weird. :((

FYI: https://getbootstrap.com/docs/4.0/layout/grid/

Reply:
Ohhhh. Thank you.